tests/test_database.py

```diff
@@ -38,31 +24,35 @@ def setUpClass(self):
         self.test_tag_manager = database_manager.TagManager(self.test_database_connector)
 
     def setUp(self):
-        create_test_database(self.test_database_connector)
-        set_test_database(self.test_database_connector)
-        create_items_test_table(self.test_database_connector)
-        create_tags_test_table(self.test_database_connector)
+        create_test_database_setup(self.test_database_connector)
     
     def test_add_item(self):
-        count = self.test_database_connector.execute("SELECT COUNT(*) FROM items;")
+        count = self.test_database_connector.execute("SELECT COUNT(*) FROM Items;")
         self.assertEqual(count['COUNT(*)'], 0)
         self.test_item_manager.add_item()
-        count = self.test_database_connector.execute("SELECT COUNT(*) FROM items;")
+        count = self.test_database_connector.execute("SELECT COUNT(*) FROM Items;")
         self.assertEqual(count['COUNT(*)'], 1)
     
     def test_delete_item(self):
-        count = self.test_database_connector.execute("SELECT COUNT(*) FROM items;")
+        count = self.test_database_connector.execute("SELECT COUNT(*) FROM Items;")
         self.assertEqual(count['COUNT(*)'], 0)
         item_id = self.test_item_manager.add_item()
-        count = self.test_database_connector.execute("SELECT COUNT(*) FROM items;")
+        count = self.test_database_connector.execute("SELECT COUNT(*) FROM Items;")
         self.assertEqual(count['COUNT(*)'], 1)
         self.test_item_manager.delete_item(item_id)
-        count = self.test_database_connector.execute("SELECT COUNT(*) FROM items;")
+        count = self.test_database_connector.execute("SELECT COUNT(*) FROM Items;")
         self.assertEqual(count['COUNT(*)'], 0)
+    
+    def test_add_tag(self):
+        count = self.test_database_connector.execute("SELECT COUNT(*) FROM Tags;")
```

Comment:
1. Add an assertTagsCount() helper function in this class.
2. Restructure tests into Arrange Act Assert
3. Skip testing during setup, if you need to trust a function add another test for it.

Reply:
As discussed preciously, this will be fixed in V2